_includes/methods/markdown/privacy.md

```diff
@@ -0,0 +1,26 @@
+## How to do it
+
+  1. Familiarize yourself with the [Fair Information Practice Principles](https://www.fpc.gov/resources/fipps/), a set of precepts at the heart of the U.S. Privacy Act of 1974.
+  1. Consult your organization's privacy office, which may include your general counsel, if you plan to substantially make use of information that could potentially identify specific individuals.
+  1. Inform and collect the voluntary consent of anyone who participates in moderated design research. Ensure that all unmoderated forms of research (for example, web analytics) are covered by an easy-to-access privacy policy.
+  1. Pay special attention to all categories of information used throughout the design process. Note contexts in which it's not okay to share certain categories of information.
+
+<section class="method--section method--section--additional-resources" markdown="1">
+
+## Additional resources
+
+- [Doing research at TTS — TTS Handbook](https://handbook.18f.gov/research-guidelines/)
+- [Privacy — TTS Handbook](https://handbook.tts.gsa.gov/launching-software/privacy)
+</section>
+
+<section class="method--section method--section--government-considerations" markdown="1" >
+
+## Considerations for use in government
+
+The government's use of information about people is subject to a number of laws and policies, including: <a href="https://www.justice.gov/opcl/overview-privacy-act-1974-2020-edition" class="usa-link">the Privacy Act of 1974</a>, the Federal Information Security Management Act of 2002, and the eGovernment Act of 2002.
```

Comment:
[suggest] I've been redoing these types of links with markdown just to keep everything relatively similar.  Ultimately, it doesn't really matter so your call!